corehq/apps/accounting/const.py

```diff
@@ -0,0 +1,22 @@
+DEFAULT_ACCOUNT_FORMAT = 'Account for Project %s'
+EXCHANGE_RATE_DECIMAL_PLACES = 9
+
+UNLIMITED_FEATURE_USAGE = -1
+MINIMUM_SUBSCRIPTION_LENGTH = 30
+
+MAX_INVOICE_COMMUNICATIONS = 5
+DEFAULT_DAYS_UNTIL_INVOICE_DUE = 30
+SMALL_INVOICE_THRESHOLD = 1
+UNPAID_INVOICE_THRESHOLD = 1
+
+OVERDUE_INVOICE_LIMIT_DAYS = 15
+DAYS_PAST_DUE_TO_TRIGGER_DOWNGRADE = OVERDUE_INVOICE_LIMIT_DAYS + 1
+DAYS_PAST_DUE_TO_TRIGGER_DOWNGRADE_WARNING = OVERDUE_INVOICE_LIMIT_DAYS - 2
+DAYS_PAST_DUE_TO_TRIGGER_OVERDUE_NOTICE = OVERDUE_INVOICE_LIMIT_DAYS - 7
+
+LOGO_FILENAME = 'corehq/apps/accounting/static/accounting/images/Dimagi-Deep-Purple-Standard-Logo.jpg'
+PDF_LIGHT_GRAY = (0.9, 0.9, 0.9)
+PDF_STROKE_COLOR = (0.85, 0.85, 0.85)
+PDF_BLACK = (0, 0, 0)
+PDF_DEFAULT_FONT_SIZE = 10
+PDF_SMALL_FONT_SIZE = 8
```

Comment:
Can we try to reserve const files for "general purpose" consts, and leave single-use constants within the module that use them?

Reply:
https://github.com/dimagi/commcare-hq/pull/35932/commits/5aa8cce49052184e52a641eebd3cc216f5ecceb6